modules/machine-health-checks-resource.adoc

```diff
@@ -36,7 +36,7 @@ spec:
 <2> Specify a label for the machine pool that you want to check.
 <3> Specify the machine set to track in `<cluster_name>-<label>-<zone>` format. For example, `prod-node-us-east-1a`.
 <4> Specify the timeout duration for a node condition. If a condition is met for the duration of the timeout, the machine will be remediated. Long timeouts can result in long periods of downtime for a workload on an unhealthy machine.
-<5> Specify the amount of unhealthy machines allowed in the targeted pool. This can be set as a percentage or an integer.
+<5> Specify the amount of machines allowed to be concurrently remediated in the targeted pool. This can be set as a percentage or an integer. maxUnhealthy prevents further remediation if the cluster is already partially unhealthy.
```

Comment:
I'd suggest grabbing this upstream snippet to use as the second line explanation (instead of the "maxUnhealthy prevents further ..." part):
```
If the number of unhealthy machines exceeds the limit set by `maxUnhealthy`, remediation is not performed.
```